kafka/provider.go

```diff
@@ -114,6 +114,9 @@ func Provider() terraform.ResourceProvider {
 			"kafka_topic": kafkaTopicResource(),
 			"kafka_acl":   kafkaACLResource(),
 		},
+		DataSourcesMap: map[string]*schema.Resource{
+			"kafka_topic_config": kafkaTopicConfigDataSource(),
```

Comment:
shouldn't this be called `kafka_topic`?  One of the subfields is the actual config.

Reply:
@manjeetvk yeah, can we rename this to `kafka_topic`